make_ryf.py

```diff
@@ -32,13 +39,19 @@
 
     ds = {}
 
-    for var in variables: 
+    for var in variables:
         print var
         for y in (year1, year2):
-            files = glob(os.path.join(jradir,"{}.{}.*.nc".format(var,y)))
+            if jra55v1p4:
+            # see https://raw.githubusercontent.com/COSIMA/1deg_jra55_iaf/2d6fdf53ae89124e7e11d40176813c286a8279bb/atmosphere/forcing.json
+                files  = glob(os.path.join(jradir,"atmos/3hr/{v}/gr/v20190429/{v}/{v}*{yr}*.nc".format(v=var, yr=y)))
```

Comment:
I think the version should be stored in a variable, to make it easy to change, but also explicit that this is something that might need changing. 

As far as I can see it should be sufficient to use a single `glob` as I don't think the variables exist in more than one of those temporal subdirectories. e.g. something like
```
       files = glob("*/*/{var}/*/{version}/{var}/{var}*{year}*.nc".format(var=var, year=y, version=version)
```
Should probably be a test in there to make sure it is length 1, otherwise abort as there is a logic error, or the directory structure has changed.

Reply:
I was avoiding assuming that a variable would only be in one subdirectory, but yes a single grep would be more streamlined. If you'd prefer that then I'm happy for you to make this change (and the rest).